### ISSUE TITLE
getPersonGroupTrainingStatusWithPersonGroupId/MPOTrainingStatus is out of date with the Face API

### ISSUE THREAD

**Issue by NateRickard** [Microsoft/Cognitive-Face-iOS#30]
[API docs for "Get Person Group Training Status"](https://westus.dev.cognitive.microsoft.com/docs/services/563879b61984550e40cbbe8d/operations/563879b61984550f30395247)

`getPersonGroupTrainingStatusWithPersonGroupId` should be looking for these fields, per the above linked docs:

`status`
`createdDateTime`
`lastActionDateTime `
`message`

Instead, it's [attempting to read non-existent result fields](https://github.com/Microsoft/Cognitive-Face-iOS/blob/master/Pod/Classes/MPOTrainingStatus.m):

```
if (self) {
        self.personGroupId = dict[@"personGroupId"];
        self.status = dict[@"status"];
        self.startTime = dict[@"startTime"];
        self.endTime = dict[@"endTime"];
    }
```

**Comment by huxuan** [comment]
Thanks for the reporting, we will fix it soon.

**Comment by huxuan** [comment]
Fixed by #31 and enjoy pod version 1.3.2.
